cl/search/signals.py

```diff
@@ -130,3 +137,26 @@
     AudioDocument,
     oa_field_mapping,
 )
+
+
+@receiver(post_save, sender=RECAPDocument)
+def handle_recap_doc_change(sender, instance: RECAPDocument, **kwargs):
```

Comment:
I think we could add `update_fields` with `None` as a default value here:

```suggestion
def handle_recap_doc_change(sender, instance: RECAPDocument, update_fields=None, **kwargs):
```

And I think it'd be fine to remove the line below:
`update_fields = kwargs.get("update_fields", [])`

Reply:
I think the test is failing because `update_fields` now defaults to `None`, so this line needs to be removed.

`update_fields = kwargs.get("update_fields", [])`

Reply:
@albertisfu Fixed